.github/workflows/test.yml

```diff
@@ -203,76 +203,42 @@ jobs:
         id: create-instance
         if: ${{ steps.changed-files-specific.outputs.any_changed == 'true' || github.event.inputs.regenerate-disks == 'true' || github.event_name == 'push'}}
         run: |
-          gcloud compute instances create-with-container "regenerate-disk-${{ env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }}" \
+          gcloud compute instances create-with-container "sync-to-checkpoint-${{ env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }}" \
           --boot-disk-size 100GB \
           --boot-disk-type pd-ssd \
-          --create-disk name="zebrad-cache-${{ env.GITHUB_SHA_SHORT }}-${{ env.NETWORK }}-checkpoint",size=100GB,type=pd-ssd \
```

Comment:
Functional Change: This change drops the Zcash network from the disk name.

That's ok for now, but I'd like to document that the `NETWORK` env var can't be changed:
- the tests and images only work on mainnet
- a TODO for ticket #1791 (or a new ticket for running tests on testnet)

Or we could just remove the network env var entirely.

Reply:
I know this is confusing, but this change only drops the `NETWORK` from the disk used by the instance, not by the image (which is the one holding the cache). The image is built here https://github.com/ZcashFoundation/zebra/pull/4252/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R278

- I've assigned myself to: https://github.com/ZcashFoundation/zebra/issues/1791 and also added to the next sprint, to fix all issues related to Testnet (this is not one, but there are other places with a hardcoded Mainnet)

Let's track that ticket to solve this in another PR.

Reply:
As this is not required (at least in this PR) I'm reducing the diff

Reply:
> I've assigned myself to: #1791 and also added to the next sprint

We can't do reliable testnet tests, because we decided not to fix bugs that are blocking them. So that ticket should have been closed. (I closed it.)

Right now our top priority is getting CI working, and getting the lightwalletd tests working in CI.

Would it help if I stopped mentioning unrelated bugs?